tests/detect-pcre/detect-pcre-03/test.rules

```diff
@@ -0,0 +1,3 @@
+alert tcp any any -> any any (msg:"HTTP TEST"; pcre:"/^gEt/i"; pcre:"/\\/two\\//U"; pcre:"/GET \\/two\\//"; pcre:"/\\s+HTTP/R"; sid:1;)
```

Comment:
hmm I see in the C tests that this should have alerted?
```
FAIL_IF_NOT(PacketAlertCheck(p, 1) == 1);
```
I'm wondering if it has to do with the regex and escape sequences. Don't know either well. @catenacyber could you please check if this rule is correct?

Reply:
Figured it out w the help from Victor, the escape sequence is incorrect. `\\` is C's escape sequence, so, if you change the rule to 
```
alert tcp any any -> any any (msg:"HTTP TEST"; pcre:"/^gEt/i"; pcre:"/\/two\//U"; pcre:"/GET \/two\//"; pcre:"/\s+HTTP/R"; sid:1;)
```
it works as expected :)

Reply:
Ok. Thank you